packaging/docker/samples/golang/app/Dockerfile

```diff
@@ -31,11 +31,9 @@ RUN apt-get update
 RUN apt-get install -y --no-install-recommends ca-certificates dnsutils
 
 ARG FDB_WEBSITE=https://www.foundationdb.org
-RUN wget "${FDB_WEBSITE}/downloads/${FDB_VERSION}/ubuntu/installers/foundationdb-clients_${FDB_VERSION}-1_amd64.deb"
+RUN wget "https://github.com/apple/foundationdb/releases/download/${FDB_VERSION}/foundationdb-clients_${FDB_VERSION}-1_amd64.deb"
```

Comment:
Any reason not to remove the build arg `FDB_WEBSITE` if we don't use it anymore?